payments/schema/types.py

```diff
@@ -283,3 +283,8 @@ class OrderDetailsType(graphene.ObjectType):
 class FailedOrderType(graphene.ObjectType):
     id = graphene.ID(required=True)
     error = graphene.String()
+
+
+class FailedOfferType(graphene.ObjectType):
+    id = graphene.ID(required=True)
+    error = graphene.String()
```

Comment:
Would it make sense to abstract this to a generic `Error` type? E.g:
```python
class GenericErrorType(graphene.ObjectType):
    id = graphene.ID(required=True)
    error = graphene.String()

class FailedOrderType(GenericErrorType):
    pass

class FailedOfferType(GenericErrorType):
    pass
```

Reply:
good idea